en/docs/concepts/authorization/introspection.md

```diff
@@ -0,0 +1,45 @@
+# OAuth 2.0 token introspection
+
+### Usage of OAuth 2.0 Token Introspection
+OAuth 2.0 Token Introspection defines a method that allows authorized protected resources to query the authorization server
+to determine the set of metadata for a given token (access token, authorization code or a refresh token) that was presented to them by 
+an OAuth Client. Based on the metadata of the particular token, it will allow the resource server to consume the 
+protected resource.
+ 
+
+This metadata includes 
+
+- Whether the token is currently active (or if it has expired or otherwise been revoked)
+- The rights of accessing the token carries (usually conveyed through OAuth 2.0 scopes)
+- The authorization context in which the token was granted (including who authorized the token and which client it was issued to)
+
+Token introspection allows a protected resource to query this information regardless of whether it is carried in the token itself.
+
+### The states and descriptions of tokens
+
+**- Authorization codes:**
+
+
+a. `ACTIVE - ` Valid and yet to be exchanged for an access token.
```

Comment:
there is a space before the  hyphen (-) in this point but not in the others...let's make it consistent